ra/ra.go

```diff
@@ -1892,19 +1883,17 @@ func (ra *RegistrationAuthorityImpl) createPendingAuthz(ctx context.Context, reg
 		Expires:        &expires,
 	}
 
-	if identifier.Type == core.IdentifierDNS && !features.Enabled(features.VAChecksGSB) {
-		isSafeResp, err := ra.VA.IsSafeDomain(ctx, &vaPB.IsSafeDomainRequest{Domain: &identifier.Value})
-		if err != nil {
-			outErr := berrors.InternalServerError("unable to determine if domain was safe")
-			ra.log.Warningf("%s: %s", outErr, err)
-			return nil, outErr
-		}
-		if !isSafeResp.GetIsSafe() {
-			return nil, berrors.UnauthorizedError(
-				"%q was considered an unsafe domain by a third-party API",
-				identifier.Value,
-			)
-		}
+	isSafeResp, err := ra.VA.IsSafeDomain(ctx, &vaPB.IsSafeDomainRequest{Domain: &identifier.Value})
```

Comment:
The sense on this is backwards: VAChecksGSB is enabled, so we should delete this code from the RA.

Reply:
D'oh, was somewhat on auto-pilot here and missed the `!`.